client/jetpack-cloud/sections/partner-portal/stored-credit-card/index.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import React, { ReactElement } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { PaymentLogo } from '@automattic/composite-checkout';
+import EllipsisMenu from 'calypso/components/ellipsis-menu';
+import PopoverMenuItem from 'calypso/components/popover/menu-item';
+import Gridicon from 'calypso/components/gridicon';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export default function StoredCreditCard( props ): ReactElement {
+	const translate = useTranslate();
+	const creditCard = props.card;
+
+	const expiry = new Date( creditCard.expiry );
+	const month = expiry.getMonth();
+	const year = expiry.getFullYear().toString().substr( 2, 2 );
+
+	return (
+		<div className="stored-credit-card">
+			<div className="stored-credit-card__header">
+				<div className="stored-credit-card__header-left">
+					<PaymentLogo brand={ creditCard.card_type } isSummary={ true } />
+				</div>
+
+				<div className="stored-credit-card__header-right">
+					<Gridicon icon="checkmark" size={ 18 } />
+					<div className="stored-credit-card__primary">
+						<span className="stored-credit-card__primary-text">{ translate( 'Primary' ) }</span>
+					</div>
+					<EllipsisMenu className="stored-credit-card__ellipsis" position="bottom">
+						<PopoverMenuItem>{ translate( 'Make primary' ) }</PopoverMenuItem>
+						<PopoverMenuItem>{ translate( 'Update details' ) }</PopoverMenuItem>
+						<PopoverMenuItem>{ translate( 'Delete card' ) }</PopoverMenuItem>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'Delete Card', { context: 'Button label'} )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jetpack-partner-portal-payment-method-list)